examples/compaction_analysis.py

```diff
@@ -59,8 +55,10 @@
     "in meters": True,
```

Comment:
Slightly unrelated to this particular instance, but if we would like to go for units i am not sure if boleans are the way to go.

Reply:
No they are not. This flag is in fact not important anymore as the code is not affected by it. I think what was more important than the unit was that width and height were provided in metric lengths and not pixels. But this does not matter anymore.